src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawing {
+    protected String color;
+    protected double area;
```

Comment:
remove this field, you don't need it, such value should be returned directly from the method for calculation of the area